### PR TITLE
updated info about integrations in subscription settings

### PR DESCRIPTION
**Pull request by abhijitkini**

### CODE REVIEW

**Comment by abhijitkini** [comment]
https://github.com/Shippable/docsv2/issues/407

File: sources/ci_configure.md [hunk 1]
```diff
@@ -154,19 +154,24 @@ build:
...
 
-You need to add a hub integration to your project settings if you want to do the following:
+You need to add a hub integration to your project, if you want to do the following:
 
 - Pull an image from a private Docker repository
 - Build a Docker image which has a `FROM` that pulls an image from a private Docker repository
 - Push to a Docker repository
 
-You can add a Hub integration to project settings by going to your Project Settings UI and navigating to the `Integrations` section. Click on the `Hub integration` dropdown to select an existing integration or create a new one.  
+To [add a Hub integration](ci_subscriptions/#intergrations) to a project do the following:
```
Comment (a-murphy) [inline sources/ci_configure.md]: Does this link work?

File: sources/ci_configure.md [hunk 2]
```diff
@@ -154,19 +154,24 @@ build:
 The image you specify in this section should be available to Shippable when the build reaches this step. To learn how to build your CI image from a Dockerfile or pull from a registry, check out the sections below.
 
 <a name="add_project_integrations"></a>
-###Adding a hub integration to project settings
+###Adding a hub integration to a project
 
-You need to add a hub integration to your project settings if you want to do the following:
+You need to add a hub integration to your project, if you want to do the following:
```
Comment (stephanielingwood) [inline sources/ci_configure.md]: Awkward comma. What about something like:
To do the following, add a hub integration to your project:
or
Hub integrations must be added to projects to do any of the following:

or something along those lines

File: sources/ci_configure.md [hunk 3]
```diff
@@ -999,7 +1004,7 @@ If you need help defining secure variables, you can check out [our instructions]
 
 ## Notifications
 
-Shippable supports email, Slack, HipChat and IRC notifications and these can be configured in your yml file.
+Shippable supports email, Slack, HipChat and IRC notifications and to enable them, it should configured in your `shippable.yml` file for the project. For Slack and HipChat, the integrations need to be added in the UI (Subscription Settings, in addition the `shippable.yml`.
```
Comment (a-murphy) [inline sources/ci_configure.md]: How about "to enable them, configure them in your `shippable.yml` file for the project."?

File: sources/ci_configure.md [hunk 4]
```diff
@@ -1267,8 +1274,14 @@ Shippable performs the following steps for you, to deploy your source code on EB
...
-<img src="../images/project_settings_deploy_integration_aws.png" alt="Account Settings Subscription" style="width:400px;"/>
+1. **Add Amazon AWS integration to the project**: 
+     - Go to the 'Settings' tab of your subscription.
+     - Under the 'Integrations' section, click the 'Add Integration' button.
+     - In the dropdown of the 'Account Integrations' field, select 'AWS'. 
+     - If you have created the AWS integration before, it'll show up as an option in the dropdown. Select AWS. This enables Shippable to authenticate into AWS. Given below is a screen shot of the Subscription Settings page, where 'AWS' is being added to the subscription 'Shippable-Demo'. 
+     - If you don't see an option of AWS in the dropdown, instructions on setting up Amazon EB account integration in Shippable can be found [here](int_paas_iaas_providers.md).
```
Comment (a-murphy) [inline sources/ci_configure.md]: "an AWS option" or "an option for AWS" or just "AWS"

File: sources/ci_projects.md [hunk 5]
```diff
@@ -173,20 +173,6 @@ Select the branch you want the status for and pick between the 'Image URL' or 'M
...
-1. Through **Project Settings**: You can create an integration for the first time through the 'Settings' tab of your project. In the 'Settings' tab of your project, select the required integration dropdown & click 'Create Integration' as shown below.
-<img src="../images/project_settings_integration.png" alt="Select Account Integration" style="width:700px;"/>
-Proceed with the configuration of a specific integration. Once you create this integration, it'll be available for your projects.
-
-2. Through **Account Settings**: Alternatively, you can enable an integration by following the [instructions here](int_overview.md#adding-an-account-integration).
-
-**IMPORTANT**: After creating the 'Integrations' in the UI, you'll need to configure the same in the `shippable.yml` file. More details on [yml config here](ci_configure.md). The same is true for any Docker registry integrations.
-
-Instructions on specific account integrations are in our [Integrations section](int_overview.md)
+To add an integration to your project, go to the Subscription's 'Settings'. [Click here](ci_subscriptions/#intergrations) for instructions.
```
Comment (a-murphy) [inline sources/ci_projects.md]: Another "intergrations".

File: sources/ci_configure.md (hunk 1 above)
Comment (abhijitkini) [inline sources/ci_configure.md]: Fixed the typo and ensured the link works.

File: sources/ci_configure.md (hunk 2 above)
Comment (abhijitkini) [inline sources/ci_configure.md]: Changed it to "Add a hub integration to your project to do any of the following:"

File: sources/ci_configure.md (hunk 3 above)
Comment (abhijitkini) [inline sources/ci_configure.md]: Changed it to "and to enable them, configure the integration in your `shippable.yml` file"

File: sources/ci_configure.md (hunk 4 above)
Comment (abhijitkini) [inline sources/ci_configure.md]: Changed it to: "If you don't see an AWS option in the dropdown"

Comment (a-murphy) [inline sources/ci_configure.md]: Did you miss a file in your updated commit?

File: sources/ci_subscriptions.md [hunk 6]
```diff
@@ -142,14 +103,119 @@ To select your Machine Image, Go to the 'Settings' tab of your 'Subscription'. C
...
 
-
-### Deployment key
+#### Deployment key
 The Deployment key section shows the SSH public key associated with your Shippable Account. You will need this key to deploy to cloud providers that supports git based deployments like Heroku and Red Hat Openshift. This key is also used to encrypt any environment variables that you want to use during your build.
 
 Our How To guides provide instructions on how to enable continuous deployment to different providers.
 
+#### Technical Contact
+Do ensure this field is populated with a valid email address for the subscription. This email will be used to notify information on service upgrades, service performance and other related topics. It will not be used for marketing purpose.
```
Comment (a-murphy) [inline sources/ci_subscriptions.md]: "marketing purposes"

**Pull request merged**